3 - PB/Documentazione interna/Verbali/24-04-15/tasks.json

```diff
@@ -0,0 +1,42 @@
+[
+	{
+	  "ID task": "DOC-699",
+	  "Descrizione": "Aggiornare requisiti di sistema manuale utente.",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "ID task": "DOC-707",
+	  "Descrizione": "Aggiungere glossario manuale utente.",
+	  "Ruolo": "Amministratore"
+	},
+	{
+	  "ID task": "DOC-709",
+	  "Descrizione": "Revisione generale del documento Norme di Progetto.",
+	  "Ruolo": "Verificatore"
+	},
+	{
+	  "ID task": "DOC-711",
```

Comment:
non credo sia ragionevole evidenziare modifiche a documenti in corso di valutazione del prof